plugins/module_utils/network/iosxr/rm_templates/bgp_neighbor_address_family.py

```diff
@@ -673,6 +673,66 @@ def __init__(self, lines=None):
                 }
             }
         },
+        {
+            "name": "route_policies.inbound",
```

Comment:
Will `route_policy` still be rendered in facts?

Reply:
no it won't as in running config there is always "route-policy route-policy-name in/out" this line will present.

Reply:
Then we can remove `route_policy` parser? Also, what happens if someone sets `route_policy` option in task?